content/blog/2022/12/2022-12-05-jenkins-newsletter.adoc

```diff
@@ -0,0 +1,166 @@
+---
+layout: post
+title: "Jenkins November 2022 Newsletter"
+tags:
+- jenkins
+- newsletter
+- community
+- contribute
+authors:
+- alyssat
+opengraph:
+  image: /images/post-images/2022-10-13-jenkins-newsletter/centered-newsletter.png
+links:
+  discourse: true
+---
+
+== November 2022
+
+image:/images/post-images/2022-10-13-jenkins-newsletter/centered-newsletter.png[Jenkins September Newsletter]
+
+Welcome to the Jenkins Newsletter! This is a compilation of progress within the project, highlighted by Jenkins Special Interest Groups (SIGs) for the month of November.
+
+Got Inspiration? We would love to highlight your cool Jenkins innovations.
+Share https://docs.google.com/forms/d/e/1FAIpQLScMCGOMtn2hGpfXsbyssGhVW1LwlW4LkXCIaKINKDQU2m6ieg/viewform[your story] and you could be in the next Jenkins newsletter.
+View previous editions of the Jenkins Newsletter link:/node/tags/newsletter/[*here*]!
+
+Happy reading!
+
+== Highlights:
+
+* Congratulations to the new officers and members of the governance
+board.
+* Pipeline Utility Steps Plugin was impacted by
+https://commons.apache.org/proper/commons-configuration/security.html#CVE-2022-33980_prior_to_2.8.0.2C_RCE_when_applied_to_untrusted_input[CVE-2022-33980],
+leading to a remote code execution in Jenkins.
+* New “jenkinsci” NPM official account for Jenkins project.
+* ssh-agent docker image is now using a https://github.com/jenkinsci/docker-ssh-agent/commit/9abba2cf2b286ad35bdb0df634c3e471da373478[correct volume] for agent workspace.
+* Jenkins 2.375.1 was released with more improvements to the user interface.
+* Jenkins.io is now utilizing Algolia v3 for its search feature.
+* Jenkins in Google Summer of Code 2023, preparations to help new contributors prepare effective proposals.
+
+== Governance Update
+
+Contributed by: https://github.com/MarkEWaite[Mark Waite]
+
+The 2022 Jenkins elections are complete.
+Thanks to Kevin Martens for the https://www.jenkins.io/blog/2022/11/17/jenkins-election-candidates-and-results/[blog post] announcing the results of the election and thanks to Damien Duportal for running the election process.
+We are glad to have received 65 registrations to vote in the election.
+That is less than the 81 registered voters we had in 2021.
+
+Elected officers and members of the Jenkins governance board will begin their service December 3, 2022.
+Special thanks to the members of the governance board that have completed their two-year service:
+
+* https://www.jenkins.io/blog/authors/halkeye/[Gavin Mogan]
+* https://www.jenkins.io/blog/2021/03/22/governance-updates/#about-ewelina-wilkosz[Ewelina Wilkosz]
+
+Congratulations to the new members of the governance board.
+
+* https://www.jenkins.io/blog/authors/uhafner/[Ullrich Hafner]
+* https://www.jenkins.io/blog/authors/notmyfault/[Alexander Brandes]
+
+Congratulations also to the newly elected officers:
+
+* Release Officer - https://www.jenkins.io/blog/authors/timja/[Tim Jacomb]
```

Comment:
shouldn't this be like `link:/blog/authors/timja/` or something that mark always suggests?

Reply:
I think it should be this:
```suggestion
* Release Officer - author:timja[]
```